tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -640,8 +729,8 @@ def insert_delay(mon_dep):
     logger.info(f"Updating initialDelaySeconds on deployment: {mon_dep}")
     kubeconfig = config.RUN.get("kubeconfig")
     cmd = (
-        f"oc get --kubeconfig {kubeconfig} deployment {mon_dep} -o yaml | "
-        f'sed "s/initialDelaySeconds: 10/initialDelaySeconds: 10000/g" | oc replace -f - '
+        f"oc --kubeconfig {kubeconfig} get deployment {mon_dep} -o yaml | "
+        f'sed "s/initialDelaySeconds: 10/initialDelaySeconds: 10000/g" | oc --kubeconfig {kubeconfig} replace -f - '
     )
     logger.info(f"Executing {cmd}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)